process/block/baseProcess_test.go

```diff
@@ -314,24 +314,38 @@ func (wr *wrongBody) IsInterfaceNil() bool {
 	return false
 }
 
+func CreateMockArguments() blproc.ArgsShardProcessor {
+	arguments := blproc.ArgsShardProcessor{
+		ArgsBaseProcessor: &blproc.ArgsBaseProcessor{
```

Comment:
Should instantiate the fields with the most used variants as seen in shardBlock_test.go

Reply:
Done.